radix-engine-stores/Cargo.toml

```diff
@@ -9,6 +9,7 @@ utils = { path = "../utils", default-features = false }
 radix-engine-common = { path = "../radix-engine-common", default-features = false }
 radix-engine-derive = { path = "../radix-engine-derive", default-features = false }
 radix-engine-store-interface = { path = "../radix-engine-store-interface", default-features = false }
+paste = { version = "1.0.13" }
```

Comment:
Oh? I'm surprised this is needed?

My bad! I think perhaps the crate with the macro definition (sbor?) needs changing to externalize the paste crate, by eg adding the following to the `sbor` crate's `lib.rs`:

`pub extern crate paste;`

And then we _may_ need to update the `paste!` invocations inside the versioned macro definitions with `paste::paste!` or `crate::paste::paste!`... I can't quite remember how this works.

Anyway, if we do it right, the macro should be self-consistent, and we should be able to remove `paste` from the above.

Reply:
I now tried a few combinations, and the compiler always complained about something not found.

I agree it would be cool to have the macro self-contained, but probably I - a macro noob - am not up to this task :(
(BTW I see `paste = { version = "1.0.13" }` in more places, so maybe this/all macros need a wider fix anyway)

Reply:
alright, I finally did that 🎉